source/documentation/runbooks.html.md.erb

```diff
@@ -1,37 +1,56 @@
-* [Add GitHub collaborators from a fork PR](runbooks/add-collaborators-from-fork.html)
-* [Operations Engineering Communications Plan](runbooks/ops-eng-communications-plan.html)
-* [LastPass - Add new user](runbooks/add-lastpass-user.html)
-* [LastPass - Disable MFA](runbooks/Lastpass-mfa.html)
-* [LastPass - Reset User's Master Password](runbooks/lastpass-password-reset.html)
-* [Create a Sentry Internal Integration](runbooks/sentry-internal-integrations.html)
-* [AWS Credentials Remediation Process] (runbooks/aws-credentials.html)
-* [New Joiners Guide](runbooks/new-joiners.html)
-* [Leavers Guide](runbooks/leavers.html)
-* [Add a maintenance-page/decommission page](runbooks/add-a-maintenance-page.html)
-* [Add user to Docker](runbooks/add-docker-user.html)
-* [Add GitHub User to MoJ Organization](runbooks/add-github-user.html)
-* [Automatic archiving of GitHub repositories](runbooks/repository-archiving.html)
-* [Branch Protection Setting and Issues](runbooks/branch-protection-settings-and-issues.html)
-* [Add GH and AWS OIDC](runbooks/add-oidc-github-aws.html)
-* [Supporting migrations/DNS Cutovers](runbooks/dns-cutovers.html)
-* [Adding a Slack alert to our alert channel](runbooks/add-a-slack-alert.html)
-* [CircleCI Contexts and Orbs](runbooks/circleCI.html)
-* [Repository Badge](runbooks/add-repo-badge.html)
-* [Add SSO to a tool](runbooks/add-sso-to-a-tool.html)
-* [Create An Auth0 Tenant](runbooks/create-an-auth0-tenant.html)
-* [Getting Access to PagerDuty](runbooks/get-access-to-pagerduty.html)
-* [Adding a repository to SonarCloud](runbooks/add-a-repo-to-sonarcloud.html)
-* [Compile the Operations Engineering Metrics pack](runbooks/compile-the-metrics-pack.html)
-* [BT DNS Change Process Pre-requisites](runbooks/BT_pre_requisites.html)
+Documentation supporting day to day processes in the Operations Engineering team
+
+## DNS
+
+* [Create Manual SSL Certificate Processes](runbooks/dns/manual-ssl-certificate-processes.html))
+* [Supporting migrations/DNS Cutovers](runbooks/dns/dns-cutovers.html)
+* [BT DNS Change Process Pre-requisites](runbooks/dns/BT_pre_requisites.html)
+
+## Services
+
+* [Add a maintenance-page/decommission page](runbooks/services/add-a-maintenance-page.html)
+* [Create a Sentry Internal Integration](runbooks/services/sentry-internal-integrations.html)
+* [AWS Credentials Remediation Process](runbooks/services/aws-credentials.html)
+* [OS Data Hub API Key Management](runbooks/services/os-places-api-key-management.html)
+* [Add user to Docker](runbooks/services/add-docker-user.html)
+* [CircleCI Contexts and Orbs](runbooks/services/circleCI.html)
+* [CircleCI Troubleshooting](runbooks/services/circleCI-troubleshoot.html)
+* [Add SSO to a tool](runbooks/services/add-sso-to-a-tool.html)
+* [Create An Auth0 Tenant](runbooks/services/create-an-auth0-tenant.html)
+* [Getting Access to PagerDuty](runbooks/services/get-access-to-pagerduty.html)
+
+
+### LastPass
```

Comment:
Should this be two hashes as per above and below or is it a sub heading under the Services heading?

Reply:
sub heading within services :)